packages/core/src/stylable-processor.ts

```diff
@@ -148,6 +148,9 @@ export const processorWarnings = {
     ILLEGAL_CSS_VAR_ARGS(name: string) {
         return `css variable "${name}" usage (var()) must receive comma separated values`;
     },
+    INVALID_CUSTOM_PROPERTY_AS_VALUE(name: string, as: string) {
+        return `invalid custom property rename ${name} as ${as} the "as" name must begin with "--" (double-dash)`;
```

Comment:
I would refer to the "as" as local alias and simplify the warning/error:

`invalid CSS property alias name ${name}. CSS properties must be prefixed with "--" (double-dash)`

or maybe simply reuse the `ILLEGAL_CSS_VAR_USE` message

Reply:
taking this but added the original name.